test/common/browser.js

```diff
@@ -0,0 +1,5 @@
+/*global expect:true*/
+/* eslint no-unused-vars: "off" */
+expect = window.weknowhow.expect.clone();
+expect.installPlugin(window.weknowhow.unexpectedSinon);
```

Comment:
```suggestion
expect.use(window.weknowhow.unexpectedSinon);
```

... `installPlugin` is the legacy alias.